go/processor/floria/processor.go

```diff
@@ -24,6 +24,12 @@ const (
 	TxAccessListAddressGas    = 2400
 	TxAccessListStorageKeyGas = 1900
 
+	CallValueTransferGas = tosca.Gas(9000)
+	CreateGas            = tosca.Gas(32000)
+	CreateDataGas        = tosca.Gas(200)
+	MemoryGas            = tosca.Gas(3)
+	SstoreSetGasEIP2200  = tosca.Gas(20000)
```

Comment:
If these constants are only ever to be used by the state contract (and tests for it) I would suggest to move them to the corresponding file.

Reply:
also if they are not used in `integration_tests` package, they can be private.